lib/Frontend/CompilerInvocation.cpp

```diff
@@ -632,6 +633,10 @@ static bool ParseLangArgs(LangOptions &Opts, ArgList &Args,
     Opts.EnableExperimentalStringProcessing = true;
   }
 
+  if (Args.hasArg(OPT_experimental_package_interface_load)) {
+    Opts.EnablePackageInterfaceLoad = true;
+  }
```

Comment:
Should we add an env var as well to simplify qualifications? Something like:
```
else {
  Opts.EnablePackageInterfaceLoad  = ::getenv("SWIFT_PACKAGE_INTERFACE_LOAD")
}
```

Reply:
I have another PR that handles this but could add it here instead.